### PR TITLE
tidy HTML

### PR DESCRIPTION
**Pull request by oliver-jk-redding**
Tidy HTML by adding indentations.

### CODE REVIEW

File: pristine-zen-garden.html [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+            <ul>
+              <li class="next">
+                <a href="/214/page3">
+                Next Designs <span class="indicator">&rsaquo;</span> </a>
+              </li>
+              <li class="previous"> <a href="/214/page1/">
+                <span class="indicator">&lsaquo;</span> Previous Designs              </a>
+              </li>
+              <li class="viewall">
+                <a href="http://www.mezzoblue.com/zengarden/alldesigns/" title="View every submission to the Zen Garden."> View All Designs </a>
```
Comment (danieldelacruz01) [inline pristine-zen-garden.html:163]: Nice indentation! I don't see any major problems with it.. Just lines 157-158 could probably be on one line.

Also, while this probably isn't part of the assignment, you might want to fix the weird text spacing on 132, 138, 141, and 161

**Comment by LucyMoore** [comment]
looks ok to me. Nice job

**Comment by oliver-jk-redding** [comment]
@danieldelacruz01  thanks all fixed!

**Comment by danieldelacruz01** [comment]
Hey @oliver-jk-redding is there any chance you could add me as a collaborator so I can accept your changes?

**Pull request merged**